### PR TITLE
Use emojis for hold and hold cancel

### PR DESCRIPTION
**Pull request by dims**
inspired by - https://twitter.com/LachlanEvenson/status/997207637019320320

We use a :lock: for hold and a :key: emoji for cancelling hold

### CODE REVIEW

**Comment by dims** [comment]
cc @cblecker

File: prow/plugins/hold/hold_test.go [hunk 1]
```diff
@@ -55,13 +55,27 @@ func TestHandle(t *testing.T) {
...
 		{
 			name:          "requested hold cancel",
 			body:          "/hold cancel",
 			hasLabel:      true,
 			shouldLabel:   false,
 			shouldUnlabel: true,
 		},
+		{
+			name:          "requested hold cancel",
+			body:          "/🔓",
```
[the remark this answers is not in the document]
Comment (dims) [reply prow/plugins/hold/hold_test.go]: yeah unfortunately some other services seem to have the same issue ( https://emojipedia.org/open-lock/ ) can't seem to find anything better for unlock, though there's one for locked with key - https://emojipedia.org/closed-lock-with-key/

[the remark this answers is not in the document]
Comment (BenTheElder) [reply prow/plugins/hold/hold_test.go]: what about `/🔑` like you're using a key to unlock it? 🤔

[the remark this answers is not in the document]
Comment (dims) [reply prow/plugins/hold/hold_test.go]: i like that

**Comment by MHBauer** [comment]
@carolynvs WDYT? I like this.

**Comment by cblecker** [comment]
/hold

Going to comment on this in the morning :)

**Comment by carolynvs** [comment]
While I do love emojis, my concern about something like this is that emoji are hard to search for and aren't friendly to people from across languages/cultures. What may make immediate sense to someone in on the joke, may be confusing and difficult to track down for other people.

**Comment by fejta** [comment]
Perhaps we should delay this refactor until https://github.com/kubernetes/test-infra/issues/8742 is complete?

**Comment by cblecker** [comment]
Pros:
- Lots of people like emojis (myself included)
- Emojis are fun

Cons:
- The language/cultural barriers that @carolynvs notes above
- Only one command supports these, while others don't. We are inconsistent enough with our anti-patterns
- The rest of our fun/easter egg pieces of prow are all optional plugins. Someone using prow in a more professional setting can just leave those plugins off. Thinking about prow as a software product, I'd probably want command emojis to be an gated, opt-in feature.
- The lock emoji specifically isn't clear. GitHub has a "[lock conversation](https://help.github.com/articles/locking-conversations/)" feature where only those with write access can leave comments.

**Comment by dims** [comment]
🤷 :)

**Pull request closed without merging**